packages/frontend/editor-ui/src/features/chatHub/chat.store.ts

```diff
@@ -106,9 +129,11 @@ export const useChatStore = defineStore(CHAT_STORE, () => {
 			const a = messagesGraph[first];
 			const b = messagesGraph[second];
 
-			if (a.runIndex !== b.runIndex) {
-				return a.runIndex - b.runIndex;
-			}
+			// TODO: Disabled for now, messages retried don't get this at the FE before reload
+			// TOOD: Do we even need runIndex at all?
```

Comment:
I kind of want to simplify our data model a bit, possibly dropping `runIndex` (we could just sort by createdAt / updatedAt?) `turnId` (unused) and `state` (kind of unused, tricky to maintain accurately) mechanisms